mkdocs.yml

```
@@ -106,7 +106,7 @@ nav:
             - Back Office tabs:
                 - Back Office tabs:  administration/back_office/back_office_tabs/back_office_tabs.md
                 - Create dashboard tab: administration/back_office/back_office_tabs/create_dashboard_tab.md
-            - Add tab switcher in Content Type: administration/back_office/content_tab_switcher.md
+            - Tab switcher in Content edit page: administration/back_office/content_tab_switcher.md
```

```suggestion
                - Tab switcher: administration/back_office/content_tab_switcher.md
```

I would add it under Back Office tabs as it is about tabs and shorter name fits better in TOC